tests/components/totalconnect/common.py

```diff
@@ -420,7 +435,7 @@ async def setup_platform(
     mock_entry.add_to_hass(hass)
 
     responses = [
-        RESPONSE_AUTHENTICATE,
```

Comment:
Can this const and `RESPONSE_AUTHENTICATE_FAILED ` be removed entirely now?

Reply:
Probably.  Can get it done this weekend